src/utils/window.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// responsive_images.js
+filepicker.extend('window', function(){
+
+    return {
+        getWidth:getWidth,
+        getHeight:getHeight
+    };
+
+    /**
+        * Get the window width.
+        * @returns {number}
+    */
+    function getWidth() {
+        return document.documentElement.clientWidth || document.body && document.body.clientWidth || 1024;
```

Comment:
i would suggest putting ` document.body && document.body.clientWidth` in brackets, for the code to be more readable.
``` js
return document.documentElement.clientWidth || (document.body && document.body.clientWidth) || 1024;
```